packages/extension/src/common/vscode/models/editor.ts

```
@@ -0,0 +1,20 @@
+ * End of line character preference.
+ */
+export const enum EndOfLineSequence {
+  /**
+   * Use line feed (\n) as the end of line character.
+   */
+  LF = 0,
+  /**
+   * Use carriage return and line feed (\r\n) as the end of line character.
+   */
+  CRLF = 1,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)